src/components/ReadyToStart/ReadyToStart.module.css

```diff
@@ -18,6 +18,11 @@
   margin-block: 85px;
 }
 
+.mainContent .contactLink {
+  padding-top: 8px;
+  padding-bottom: 12px;
+}
```

Comment:
Should be able to remove this after making the Button component CSS change

Reply:
I have removed the class and style from the component. The button should now be aligned properly for both. But I have left the button action/ routing for a separate ticket.

Reply:
Hello @FevenSeyfu and @joanwolk,

I have reviewed the changes made by Joan and observed that the "ready-to-get-started" element aligns well with the button in the desktop view. However, the problem still exists in the mobile view as the image is overlapping the text. I'm not sure if it has been fixed.

Reply:
Hello @joanwolk, please could you review the PR gain?

Reply:
@keziabaidoo can you please take a screenshot of what you're seeing? I don't see any overlap at any viewport width. Here are several screenshots from what I see in Firefox.

<img width="567" alt="Screenshot 2024-07-24 at 11 41 10 PM" src="https://github.com/user-attachments/assets/2a7aebcb-db69-4a8c-b580-47e5ba643357">

<img width="801" alt="Screenshot 2024-07-24 at 11 41 48 PM" src="https://github.com/user-attachments/assets/87dff0dc-355a-4297-8a51-3c6352a56a2b">

<img width="804" alt="Screenshot 2024-07-24 at 11 41 59 PM" src="https://github.com/user-attachments/assets/35f606a9-b4f9-4180-9a7d-df6beda53ac1">

<img width="1007" alt="Screenshot 2024-07-24 at 11 42 24 PM" src="https://github.com/user-attachments/assets/7f5948c0-7e68-4fa5-8c76-fbc6c39b0f43">